clients/src/main/java/org/apache/kafka/clients/admin/LogDirDescription.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients.admin;
+
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.errors.ApiException;
+
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableMap;
+
+/**
+ * A description of a log directory on a particular broker.
+ */
+public class LogDirDescription {
+    private final Map<TopicPartition, ReplicaInfo> replicaInfos;
+    private final ApiException error;
+
+    public LogDirDescription(ApiException error, Map<TopicPartition, ReplicaInfo> replicaInfos) {
+        this.error = error;
+        this.replicaInfos = replicaInfos;
+    }
+
+    /**
+     * A KafkaStorageException if this log directory is offline,
+     * possibly some other exception if there were problems describing the log directory
+     * or null if the directory is online.
+     */
+    public ApiException error() {
```

Comment:
How about this? (in consistency with current `DescribeLogDirsResponse.LogDirInfo`):

```
Returns `ApiException` if the log directory is offline or an error occurred. If not, returns null.
<p><ul>
<li> KafkaStorageException - The log directory is offline.
<li> UnknownServerException - The server experienced an unexpected error when processing the request.
</ul><p>
```

(Description of `UnknownServerException` was from `Errors.UNKNOWN_SERVER_ERROR`)

Reply:
For consistency with the other [Javadoc](https://github.com/apache/kafka/blob/trunk/streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java#L1078) it seems like we should use a single `<p>` only when between the paragraphs. Please remove the `<p>` tags. (Sorry, I was also confused.)

Reply:
We should remove the `<p>` before the `<ul>` also. (see below.)
![kafka KafkaStreams java at trunk · apache kafka](https://user-images.githubusercontent.com/2375128/87550564-23978180-c6ea-11ea-9fcf-c6a88ea3034d.png)